src/main/java/thePackmaster/patches/RenderBaseGameCardPackTopTextPatches.java

```diff
@@ -65,7 +65,7 @@ private static boolean isInPackmasterRun() {
         return AbstractDungeon.player != null && AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER;
     }
 
-    private static boolean isInPackmasterCardLibraryScreen() {
+    public static boolean isInPackmasterCardLibraryScreen() {
```

Comment:
This set of functions could be moved to some common location, now that they're being used by a few different patches.

I decided not to do that because I don't see these functions as more broadly useful. They help with a small and specific set of functionality, and I don't think there will be much more code written that would want to use this.